src/coreclr/src/jit/optimizer.cpp

```diff
@@ -7561,6 +7561,7 @@ void Compiler::fgCreateLoopPreHeader(unsigned lnum)
                         preHead->bbFlags |= BBF_JMP_TARGET | BBF_HAS_LABEL;
                     }
                 } while (++jumpTab, --jumpCnt);
+                __fallthrough; // TODO: seems like a bug
```

Comment:
@sandreenko I guess this should be break instead, otherwise the  BBJ_SWITCH always ends up in the noway_assert below, right?

Reply:
Ah, I can see @sandreenko is OOF, @dotnet/jit-contrib can someone else please answer my question?

Reply:
Yeah, it looks like it should be `break` but currently this code is unreachable, I will check why.